vendordeps/Phoenix.json

```diff
@@ -27,81 +25,55 @@
             "version": "5.20.2",
             "isJar": false,
             "skipInvalidPlatforms": true,
-            "validPlatforms": [
-                "linuxathena"
-            ]
+            "validPlatforms": ["linuxathena"]
         },
```

Comment:
Don't change any of the vendor JSON files - they should be treated as checked-in binaries.